wgpu-hal/src/dx12/adapter.rs

```diff
@@ -44,21 +44,27 @@ impl super::Adapter {
         library: &Arc<native::D3D12Lib>,
         instance_flags: crate::InstanceFlags,
     ) -> Option<crate::ExposedAdapter<super::Api>> {
+        profiling::scope!("expose adapter");
```

Comment:
this thing should live in `wgpu-core` if it doesn't already? No point in adding this scope to each backend

Reply:
Another investigation artifact.